### PR TITLE
Check for tld on end of certificate path to support multi segment tld's

### PR DESCRIPTION
**Pull request by jrbarnard**
Fixes bug #823

### CODE REVIEW

File: cli/Valet/Site.php [hunk 1]
```diff
@@ -120,11 +120,21 @@ function links()
...
+
         return collect($this->files->scandir($path))->filter(function ($value, $key) {
             return ends_with($value, '.crt');
-        })->map(function ($cert) {
+        })->map(function ($cert) use ($config) {
             $certWithoutSuffix = substr($cert, 0, -4);
-            return substr($certWithoutSuffix, 0, strrpos($certWithoutSuffix, '.'));
+            $trimToString = '.';
+
+            // If we have the cert ending in our tld stripe that tld specifically
```
Comment (drbyte) [inline cli/Valet/Site.php]: `stripe` vs `strip`?

Comment (jrbarnard) [reply cli/Valet/Site.php]: @drbyte Good catch, I'll update

Comment (drbyte) [reply cli/Valet/Site.php]: 👍

**Comment by drbyte** [comment]
Looks good to me.

**Pull request merged**